tensorflow/tools/ci_build/rel/ubuntu/cpu_arm64_pip.sh

```diff
@@ -79,6 +79,7 @@ export TF_TEST_TARGETS="${DEFAULT_BAZEL_TARGETS} \
     -//tensorflow/python/kernel_tests/array_ops:split_op_test_cpu \
     -//tensorflow/python/kernel_tests/array_ops:split_op_test_gpu \
     -//tensorflow/python/kernel_tests/control_flow:scan_ops_test_cpu \
+    -//tensorflow/python/kernel_tests/control_flow:scan_ops_test_gpu \
```

Comment:
These two lines are added by the latest merge commit, but they are not in master. Shouldn't they be under the `$TF_TEST_TARGETS` line instead?